engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1064,6 +1064,10 @@ public void orchestrateStart(final String vmUuid, final Map<VirtualMachineProfil
                 }
 
                 final VirtualMachineProfileImpl vmProfile = new VirtualMachineProfileImpl(vm, template, offering, owner, params);
+                s_logger.info(" Uefi params " + "UefiFlag: " + params.get(VirtualMachineProfile.Param.UefiFlag)
```

Comment:
Could this potentially cause any exception? Should this be only logged when custom params are passed; otherwise it may always log as null/null confuse admin/users cc @pavanaravapalli @DaanHoogland @andrijapanicsb